src/schemas/staticgiturl.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe. All rights reserved.",
+    "This file is licensed to you under the Apache License, Version 2.0 (the \"License\");",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at http://www.apache.org/licenses/LICENSE-2.0",
+    "",
+    "Unless required by applicable law or agreed to in writing, software distributed under",
+    "the License is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS",
+    "OF ANY KIND, either express or implied. See the License for the specific language",
+    "governing permissions and limitations under the License."
+  ],
+  "$id": "https://ns.adobe.com/helix/shared/staticgiturl",
```

Comment:
I don't like the duplication of git url. can you extend a schema?
or should we move the `url` below the `static` in object form?

```yaml
strains:
  default:
    code: ...
    content: ...
    static: https://github.com/aobe/project-helix.io.git/htdocs

  defaultWithAllows:
    code: ...
    content: ...
    static: 
      repository:
          repo: project-helix.io
          owner: adobe
          ref: master
          path: /htdocs
      allow: ...
      deny: ...

```

Reply:
You can extend a schema, but not when you use `"additionalProperties": false`, which we do.

Introducing a `repository` property might be the right thing to do.